File: frontend/src/app/dto/comment/comment.insert.res.dto.ts
import { CommentInsertDataResDto } from "./comment.insert.data.res.dto";

export class TicketInsertResDto{
    private _data! : CommentInsertDataResDto
	private _message! : string

    public get data(): CommentInsertDataResDto {
        return this._data;
    }

    public set data(data: CommentInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: backend/src/main/java/com/lawencon/ticketms/dto/comment/CommentInsertResDto.java
package com.lawencon.ticketms.dto.comment;

public class CommentInsertResDto {
	private CommentInsertDataResDto data;
	private String message;
	public CommentInsertDataResDto getData() {
		return data;
	}
	public void setData(CommentInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
